index.html

```diff
@@ -0,0 +1,231 @@
+              sign-ups required.
+            </p>
+            <div class="tonic-courses">
+              <ul>
+                <li><a href="#">html</a></li>
+                <li><a href="#">css</a></li>
+                <li><a href="#">javascript</a></li>
+              </ul>
+            </div>
+            <div class="btn">
+              <button class="btn-tonic" type="submit">See Project</button>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="third-card-section">
+        <div class="all-components">
+          <div class="first-image">
+            <img src="./asset/Snapshoot(2).png" alt="" />
+          </div>
+          <div class="tonic-content">
+            <h1>Multi-post <br />Stories</h1>
+            <div class="tonic-content-canopy">
+              <p>CANOPY</p>
+              <ul class="tonic-list">
+                <li>Back End Dev</li>
+                <li>2021</li>
+              </ul>
+            </div>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <div class="tonic-courses">
+              <ul>
+                <li><a href="#">html</a></li>
+                <li><a href="#">css</a></li>
+                <li><a href="#">javascript</a></li>
+              </ul>
+            </div>
+            <div class="btn">
+              <button class="btn-tonic" type="submit">See Project</button>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="fourth-card-section">
+        <div class="all-components">
+          <div class="first-image">
+            <img src="./asset/Snapshoot(3).png" alt="" />
+          </div>
+          <div class="tonic-content">
+            <h1>TONIC</h1>
+            <div class="tonic-content-canopy">
+              <p>CANOPY</p>
+              <ul class="tonic-list">
+                <li>Back End Dev</li>
+                <li>2021</li>
+              </ul>
+            </div>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <div class="tonic-courses">
+              <ul>
+                <li><a href="#">html</a></li>
+                <li><a href="#">css</a></li>
+                <li><a href="#">javascript</a></li>
+              </ul>
+            </div>
+            <div class="btn">
+              <button class="btn-tonic" type="submit">See Project</button>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="about-myself-card">
+        <div class="all-components">
+          <!-- <div class="first-image">
+            <img src="./asset/Snapshoot Portfolio.png" alt="" />
+          </div> -->
+          <div class="tonic-content">
+            <h1>
+              About <br />
+              Myself
+            </h1>
```

Comment:
- [ ] As mentioned earlier make sure you use one `h1` tag in your entire code, kindly use other `hx` tag `h2-h6` instead👍